src/build/__tests__/build-test.js

```diff
@@ -149,4 +165,16 @@ describe('build()', () => {
       Timestamp: { serialize },
     }, undefined, false)).forEach((type, i) => expectTypesEqual(type, [Record, Timestamp][i]));
   });
+
+  it('should allow directives configuration using __schema', () => {
+    const config = { __schema: { directives: [CustomDirective] } };
+    const target = build(schemaSource + queryWithDirectiveSource, config);
+    expectSchemasEqual(target, SchemaWithDirective);
+  });
+
+  it('should not build schema with default directives', () => {
+    const config = { __schema: { directives: [CustomDirective] } };
+    const target = build(schemaSource + queryWithDirectiveSource, config);
+    expect(() => expectSchemasEqual(target, Schema)).toThrowError();
+  });
```

Comment:
Like I mentioned in my previous comment, this test case should already be covered in `buildSchema-test.js`, and if it's not then it should be moved there since `build` is only concerned with calling `buildSchema` when necessary and appropriately. The internal works of `buildSchema` should be tested in `buildSchema-test.js`.

I'm still not sure if this test case is needed. It basically checks that when it's built with the directive it doesn't equal a schema that isn't built with it. This is tested by the fact that the first test passes.

If you're trying to check for default directives (e.g., `skip` and `include`) then the test case should look differently.

Could you maybe explain in more detail?

Please check the `buildSchema` [test file](https://github.com/bloveit/graphql-utilities/blob/master/src/build/__tests__/buildSchema-test.js) and you'll see what I mean.

Basically, the only test case needed is the first one you added but it should test if the right config object is being passed to `buildSchema` (which it's already doing but the test description needs to be updated).